HW2/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="index.css">
+</head>
+<body>
+	<div class="wrapper">
+		<header class="header">
+			<div class="logo">
+				<img src="logo.png">				
+			</div>
+				<menu class="menu">
+					<li><a class="menu__link" href="#">Главная</a></li>
+					<li><a class="menu__link" href="#">Товары</a></li>
+					<li><a class="menu__link" href="#">О_нас</a></li>
+					<li><a class="menu__link" href="#">Контакты</a></li>
+				</menu>
+		</header>
+		<div class="container">
+				<menu class="aside">
```

Comment:
call it by it's sense and meaning. Not just aside, but side-menu, left-menu or smth that matter more.